### ISSUE TITLE
Crash when using Not Enough Enchantments

### ISSUE THREAD

**Issue by Auscobe** [The-Fireplace-Minecraft-Mods/TickDynamic#50]
For issues:
 I have the latest version (And like others, it's also giving the not valid statement in chat.)
All I remember about it is that whenever you load a world, it advances a couple frames at most, and then either freezes or crashes a moment later, with the launcher showing something like "Incompatible with Java Arraylist thing". You should be able to get it to work yourself, although if you can't I can provide a crashlog for you (Which I can't do rn, has to be done later if necessary)

If this is an issue with NEE, just let me know, because I would comment there, but it has no github repo and I don't think I have a curse account.

**Comment by ghost** [comment]
I second this. Modcreator of NEE and SME gave me a response about this: 

"Simple. Tick Dynamic is a Core Mod that injects a somewhat customized list that is incompatible with array list. I can't do anything about this not unless I somehow get the permission to "collaborate" with the mod author of this. In layman's term, this mod and tick dynamic are incompatible. I'll try to see if I can do anything on my end but no promises. Will be updated in NEE so not closing this." 

I hope you'll respond about this.
